main.py

```diff
@@ -1,719 +0,0 @@
-# Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
Don't delete this, go back with what you have.

Reply:
Actually @DubeySandeep told me to work on another part of the same issue.

Reply:
Okay, in that case, please copy + paste back the file otherwise you're removing this file entirely.

Reply:
@lkbhitesh07 Have you undone the changes in this file?

Reply:
Actually i've deleted the main.py file bhaiya, and don't have content to put back as per told me to do...can you guide me furthur?